src/ol_infrastructure/infrastructure/consul/Pulumi.infrastructure.consul.mitxonline.CI.yaml

```diff
@@ -1,3 +1,9 @@
 ---
 secretsprovider: awskms://alias/infrastructure-secrets-ci
 encryptedkey: AQICAHjEYf3gV5ZTnviCJcHwGnUuLzld5EuYumgMrRde6bvXUgG73PTfwclsj8dK9LvX0UduAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMrlk7j5vEC2uehJLmAgEQgDtYHAXyIixcJXOtRZHtDBnKqam2kqQILYCU1Y4ULx6Tu08HdvzzEfG6KdJWrQHDDCFmpOZEUaYZSPeVgw==
+config:
+  aws:region: us-east-1
+  consul:basic_auth_password_hash:
+    secure: v1:xrfClIzFx1pixyvt:2ItnnYGVJQpW7Ko6KPqqGCBr0h7PJxYYI8g2gAzWttiljGUQaTWTQm5dHvypFQ/2aYzOlmhG8bxN2Lss0zsbIasISiEjAqvXB16lAg==
+  environment:business_unit: mitxonline
+  environment:vpc_reference: mitxonline_vpc
```

Comment:
No `CI` in the VPC name?

Reply:
No, because it's looking for the name exported from the network stack, which is agnostic to the environment.